app/workers/publish_links_worker.rb

```diff
@@ -0,0 +1,15 @@
+class PublishLinksWorker
+  include Sidekiq::Worker
+
+  sidekiq_options retry: 5
```

Comment:
I just realised now that we already have error handling going on here: https://github.com/alphagov/govuk_sidekiq/blob/master/lib/govuk_sidekiq/sidekiq_initializer.rb#L18 so 👍